src/Task/Git/CommitMessage.php

```diff
@@ -332,4 +372,77 @@ private function getCommitMessageLinesWithoutComments($commitMessage)
             return strpos($line, '#') !== 0;
         }));
     }
+
+    private function enforceTypeScopeConventions()
+    {
+        $config = $this->getConfiguration();
+
+        if (!is_array($config['type_scope_conventions'])) {
+            return false;
+        }
+
+        if (!in_array('types', array_keys($config['type_scope_conventions']))
```

Comment:
A bit shorter:
```php
$conventionsKeys = array_keys($config['type_scope_conventions']);

return in_array('types', $conventionsKeys) || in_array('scopes', $conventionsKeys);
```

Reply:
Agreed, this looks much better thank you ...

Reply:
This should resolve this conversation https://github.com/phpro/grumphp/pull/541/commits/e2e2393936e52cc061480c8e436d54f362f8913c